descope/internal/mgmt/project.go

```diff
@@ -69,8 +69,11 @@ func (p *project) ListProjects(ctx context.Context) ([]*descope.Project, error)
 	return response.Projects, nil
 }
 
-func (p *project) ExportSnapshot(ctx context.Context) (*descope.ExportSnapshotResponse, error) {
+func (p *project) ExportSnapshot(ctx context.Context, req *descope.ExportSnapshotRequest) (*descope.ExportSnapshotResponse, error) {
 	body := map[string]any{}
+	if req.Format != "" {
```

Comment:
I think we handle `nil` req

![image](https://github.com/user-attachments/assets/76f71e1e-0ce0-4a3d-b0e8-11d322ff4ef1)


![image](https://github.com/user-attachments/assets/5b9b97a0-7ed2-46c8-9dfc-b8566bfa5b17)